src/patternfly/utilities/Sizing/examples/sizing-breakpoint-example.hbs

```diff
@@ -0,0 +1,14 @@
+{{#> display display--type="flex" display--modifier="pf-u-flex-direction-column"}}
```

Comment:
should we add examples using `.pf-u-h-*`?

Reply:
Done, thanks for the help!